OPHD/States/MapViewStateIO.cpp

```diff
@@ -285,8 +285,6 @@ void MapViewState::load(const std::string& filePath)
 
 	difficulty(stringToEnum(difficultyTable, dictionary.get("difficulty", std::string{"Medium"})));
 
-	StructureCatalogue::init(mPlanetAttributes.meanSolarDistance);
-	ProductCatalogue::init("factory_products.xml");
```

Comment:
Look up a few lines, and you'll see where `mPlanetAttributes` is initialized.

Reply:
Huh, I thought I did a call trace to make sure that wouldn't be an issue. I'll take a closer look tonight when I get home from work.